src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,30 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
 
+        Machine[] machinesWork = new Machine[] {bulldozer, excavator, truck};
+        Machine[] machinesStop = new Machine[] {bulldozer, excavator, truck};
+
+        for (Machine machine : machinesWork) {
+            work(machine);
+        }
+
+        System.out.println();
+
+        for (Machine machine : machinesStop) {
+            stop(machine);
+        }
+    }
+
+    public static void work(Machine machine) {
```

Comment:
should we create separate static methods for starting and stoping work?

Reply:
Okey, I will create separate static method for work and start